composer.json

```diff
@@ -20,7 +20,10 @@
     "prefer-stable": true,
     "require": {
         "php": "^7.4|^8.0",
-        "foxy/foxy": "^1.1@dev",
+        "npm-asset/vizuaalog--bulmajs": "^0.11.0",
+        "npm-asset/bulma": "^0.9.1",
+        "npm-asset/bulma-helpers": "^0.3.12",
+        "oomphinc/composer-installers-extender": "@dev",
```

Comment:
There's `2.0.0` available.

Reply:
yes work fine.

Reply:
I mean dependencies are better to be tied to stable versions in case these are available.

Reply:
done.